k4MarlinWrapper/k4MarlinWrapper/converters/EDM4hep2Lcio.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
```

Comment:
We discussed `pragma once` and said to prefer old-style include guards, no?

Reply:
done.